src/Components/PaymentInputField.res

```diff
@@ -20,6 +20,7 @@ let make = (
   ~placeholder="",
   ~appearance: CardThemeType.appearance,
   ~className="",
+  ~rightIconClassName="-ml-10",
```

Comment:
one place it's -ml-10 another place -ml-12

Reply:
-ml-10 is the standard size applied in every input field for icon, however the card scheme component with cobadge component requires some extra margin as it is bigger in width.